packages/sdk/src/plt/types.ts

```diff
@@ -50,7 +50,7 @@ export type TokenAccountInfo = {
 /**
  * Represents the reason for a token module operation rejection.
  */
-export type TokenModuleRejectReason = {
+export type EncodedTokenModuleRejectReason = {
```

Comment:
We rename here to align with the corresponding representation of `TokenModuleEvent`s, i.e. 
- `EncodedTokenModuleEvent` and the corresponding decoded version `TokenModuleEvent`
- `EncodedTokenModuleRejectReason` and the corresponding decoded version `TokenModuleRejectReason`